src/main/java/org/springframework/data/redis/connection/convert/Converters.java

```diff
@@ -309,9 +309,9 @@ protected static RedisClusterNode toClusterNode(String clusterNodesLine) {
 	 * @param microseconds elapsed microseconds in current second
 	 * @return
 	 */
-	public static Long toTimeMillis(String seconds, String microseconds) {
-		return NumberUtils.parseNumber(seconds, Long.class) * 1000L
-				+ NumberUtils.parseNumber(microseconds, Long.class) / 1000L;
+	public static Long toTimeMicros(String seconds, String microseconds) {
```

Comment:
Let's keep the old method and introduce a new one to not break backward compatibility.

Also, we should use `TimeUnit.MILLIS`/`MICROS` for conversion for more expressive code. It makes sense to update `toTimeMillis` as well.